core/blockchain.go

```diff
@@ -435,6 +447,36 @@ func (bc *BlockChain) flattenSnapshot(postAbortWork func() error, hash common.Ha
 	return bc.snaps.Flatten(hash)
 }
 
+// warmAcceptedCaches fetches previously accepted headers and logs from disk to
+// pre-populate [acceptedHeadersCache] and [acceptedLogsCache].
+func (bc *BlockChain) warmAcceptedCaches() {
```

Comment:
Is it necessary to warm this cache? It should do so once it receives requests, so I'm not sure adding this piece of code is worth it.

Reply:
Prewarming 8192 blocks over gRPC takes ~9s (just for context)

Reply:
We need some notion of this or I need to add items to the cache during request processing to avoid a pretty bad cold start issue on startup IMO.